src/sources/MVT.js

```diff
@@ -86,9 +86,8 @@ export default class MVT extends Base {
      *
      * @fires CartoError
      *
-     * @constructor MVT
-     * @extends carto.source.Base
      * @memberof carto.source
```

Comment:
Currently, I can't see MVT class in https://carto.com/developers/carto-vl/reference/#cartosource category, and I think we should include it.